src/libraries/System.Net.NameResolution/System.Net.NameResolution.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
```

Comment:
Can you revert unintended changes in this file?
Same applies to `System.Net.Sockets.sln` as well.

Reply:
Oh. Not again!!!! 
I will.